e2e-tests/cypress/e2e/journeys/accessibility-statement.spec.js

```diff
@@ -31,226 +31,250 @@ const {
+        });
 
-  describe('technical info', () => {
-    it('renders a heading', () => {
-      technicalInfo.heading().invoke('text').then((text) => {
-        expect(text.trim()).equal(TECHNICAL_INFO.HEADING);
+        complianceStatus.link().should('have.attr', 'href', COMPLIANCE_STATUS.GUIDLINES_LINK.HREF);
       });
-    });
 
-    it('renders an description', () => {
-      technicalInfo.description().invoke('text').then((text) => {
-        expect(text.trim()).equal(TECHNICAL_INFO.DESCRIPTION);
+      it('renders an outro', () => {
+        complianceStatus.outro().invoke('text').then((text) => {
+          expect(text.trim()).equal(COMPLIANCE_STATUS.OUTRO);
+        });
       });
     });
-  });
 
-  describe('compliance status', () => {
-    it('renders a heading', () => {
-      complianceStatus.heading().invoke('text').then((text) => {
-        expect(text.trim()).equal(COMPLIANCE_STATUS.HEADING);
+    describe('improving accessibility', () => {
+      it('renders a heading', () => {
+        improvingAccessibility.heading().invoke('text').then((text) => {
+          expect(text.trim()).equal(IMPROVING_ACCESSIBILITY.HEADING);
+        });
       });
-    });
 
-    it('renders an intro', () => {
-      complianceStatus.intro().invoke('text').then((text) => {
-        expect(text.trim()).equal(COMPLIANCE_STATUS.INTRO);
+      it('renders an description', () => {
+        improvingAccessibility.description().invoke('text').then((text) => {
+          expect(text.trim()).equal(IMPROVING_ACCESSIBILITY.DESCRIPTION);
+        });
       });
     });
 
-    it('renders a link', () => {
-      complianceStatus.link().invoke('text').then((text) => {
-        expect(text.trim()).equal(COMPLIANCE_STATUS.GUIDLINES_LINK.TEXT);
+    describe('preperation of statement', () => {
+      it('renders a heading', () => {
+        preperationOfStatement.heading().invoke('text').then((text) => {
+          expect(text.trim()).equal(PREPERATION_OF_STATEMENT.HEADING);
+        });
       });
 
-      complianceStatus.link().should('have.attr', 'href', COMPLIANCE_STATUS.GUIDLINES_LINK.HREF);
-    });
+      it('renders a list', () => {
+        preperationOfStatement.listItem1().invoke('text').then((text) => {
+          expect(text.trim()).equal(PREPERATION_OF_STATEMENT.LIST[0]);
+        });
 
-    it('renders an outro', () => {
-      complianceStatus.outro().invoke('text').then((text) => {
-        expect(text.trim()).equal(COMPLIANCE_STATUS.OUTRO);
+        preperationOfStatement.listItem2().invoke('text').then((text) => {
+          expect(text.trim()).equal(PREPERATION_OF_STATEMENT.LIST[1]);
+        });
       });
     });
   });
 
-  describe('improving accessibility', () => {
-    it('renders a heading', () => {
-      improvingAccessibility.heading().invoke('text').then((text) => {
-        expect(text.trim()).equal(IMPROVING_ACCESSIBILITY.HEADING);
-      });
-    });
+  describe('insurance', () => {
```

Comment:
Can we seperate the tests please so we have 1 file for `quote/generic-page.spec.js` and one for `insurance/generic-page.spec.js